deployment/k8s/charts/ci/default-values.yaml

```
@@ -0,0 +1,5 @@
+replicaCount: 1
+
+image:
+  repository: titiler
```

to align with the name in the CI
```suggestion
  repository: titiler-openeo
```

---

Adjusted.